tests/dummy/app/templates/docs/testing-acceptance.md

```diff
@@ -30,16 +25,24 @@ import { visit, click, currentURL } from '@ember/test-helpers';
 module('basic acceptance test', function(hooks) {
   setupApplicationTest(hooks);
 
-  test('the user can visit /admin page', async function(assert) {
+  test('the user can visit home page', async function(assert) {
     await visit('/');
     await click('.admin-menu-item');
 
-    assert.equal(currentURL(), '/admin');
+    assert.equal(currentURL(), '/');
   });
 });
 ```
 
-## Testing for Lazy Engines
+## Host Application
+
+Sometimes it is necessary to write acceptance tests on the host app because some engines have flows that through dependencies coming from host app such as `services` that can update UI states on host app or redirect to an external context by `externalRoute`. Therefore, in acceptance tests normally we try to avoid mocking external dependencies, because it's not good for acceptance tests since you are supposed to test things as close to "real life" as possible.
+
+### Eager Engines
+
+The tests are written in the same way as in a normal Ember application.
```

Comment:
We should probably link to how this is done.

Reply:
Done!